tools/get-scala-commit-sha

```diff
@@ -1,22 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I am guessing {{_site/build.xml}} is no longer important. Just noting it has a reference to this command,

```
    <exec osfamily="unix" executable="tools/get-scala-commit-sha" outputproperty="git.commit.sha" failifexecutionfails="false" />
    <exec osfamily="windows" executable="cmd.exe" outputproperty="git.commit.sha" failifexecutionfails="false">
      <arg value="/c"/>
      <arg value="tools\get-scala-commit-sha.bat"/>
      <arg value="-p"/>
    </exec>
    <exec osfamily="unix" executable="tools/get-scala-commit-date" outputproperty="git.commit.date" failifexecutionfails="false" />
```